python/lsst/ip/isr/calibType.py

```diff
@@ -527,7 +527,8 @@ def readFits(cls, filename, **kwargs):
             for k, v in table.meta.items():
                 if isinstance(v, fits.card.Undefined):
                     table.meta[k] = None
-
+        primaryHeader = dict(fits.open(filename)[0].header)
+        tableList[0].meta.update(primaryHeader)  # This retains things it shouldn't, like 'SIMPLE'.
```

Comment:
Can you use `from astro_metadata_translator import merge_headers` for this?

Reply:
Yes. Please. That gives you much more control over how you want to deal with duplicate headers.

Reply:
Done.  Will push after repeating my tests.